readme.md

```diff
@@ -62,17 +62,25 @@ body{
 All the steps for one way of doing the above have been written below, but in the wrong order. Put them in the correct order.
 
 ```
-$ touch README.md
-$ git push origin master
-$ git push origin master
-$ git remote add origin git@github.com/username/project-repo.git
-$ git remote add juan git@github.com/juan/project-repo.git
-$ git commit -m "initial commit"
+
+```
+
+> ANSWER
 $ cd project-repo
 $ git init project-repo
-$ git add .
+touch README.md
+$ git remote add origin git@github.com/username/project-repo.git
+$ git remote add juan git@github.com/juan/project-repo.git
 $ git merge juan/feature
-```
+$ git add .
+$ git commit -m "initial commit"
+$ git push origin master
+
+
+
+
+
+
 
```

Comment:
The question proposes that you push what you have up to your own repo before merging any of Juan's code in. So you final answer to this question should have more than one `git push` statement (and the first one should be preceded by `git add` and `git commit`).

Once that's done, then you can add Juan's remote, merge his code and push it to master.